jobspec/parse.go

```diff
@@ -1638,6 +1557,139 @@ func parseUpstream(uo *ast.ObjectItem) (*api.ConsulUpstream, error) {
 
 	return &upstream, nil
 }
+func parseChecks(service *api.Service, checkObjs *ast.ObjectList) error {
```

Comment:
Note to myself / other reviewers: the diff makes it hard to see but the `parseCheck` and `parseCheckRestart` functions were just relocated in the file unchanged.